userguide/storage.rst

```diff
@@ -2431,6 +2427,43 @@ The remaining settings are left at the default choices. Make sure
 replication task.
 
 
+.. _One-time Replication:
+
+One-time Replication
+^^^^^^^^^^^^^^^^^^^^
+
+One-time replications copy manually created snapshots of a dataset to
+another target dataset. This kind of replication is compatible with any
+replication :guilabel:`Transport` method. To configure a one-time
+replication, go to
+:menuselection:`Storage --> Replication Tasks`
+and click :guilabel:`Add Replication`.
+
+Enter a descriptive :guilabel:`Name`, choose the :guilabel:`Direction`,
+and configure any :guilabel:`Transport` settings.
+
+Enter the :guilabel:`Source Datasets` that have snapshots to be copied
+to the target.
+
+Enter the :guilabel:`Target Dataset` that will store the replicated
+snapshots and decide if child dataset snapshots should also be
+replicated.
+
+Do not select a periodic snapshot task for one-time replications. Enter
+a :guilabel:`Also include naming schema` to rename replicated snapshots
```

Comment:
"Enter a"?  Is the "a" extra here?

This is also confusing.  Tell what the naming schema is (bad name, maybe we should just call it a naming format?) and then say it can be used to rename snapshots, with this example doing it based on replication time. (replication time, or snapshot time?)
"The naming schema can be used to rename replicated snapshots